src/api.js

```diff
@@ -30,4 +30,14 @@ export const getComments = (review_id) => {
     .then((response)=> {
         return response.data.comments
     })
+}
+
+export const patchVotes = (review_id, num) => {
+    const patchBody = {
```

Comment:
Really like that you've sepeated the body in this request, this makes the code super readable 👍